packages/ui-app/src/AddressInfo.tsx

```diff
@@ -172,7 +172,7 @@ class AddressInfo extends React.PureComponent<Props> {
             data-for={`controlled-trigger${index}`}
           />
           <Tooltip
-            text={`${remainingBlocks} blocks left`}
+            text={t(`${remainingBlocks} blocks left`)}
```

Comment:
Should be `t('{{remainingBlocks}} blocks left', { replace: { remainingBlocks } })` - yes, I know copied, however the ui-reactive stuff doesn't have translation.